website_authorized_service_centers/views/authorized_service_centers.xml

```diff
@@ -16,7 +16,7 @@
                     </section>
                     <section class="mt16 mb16">
                         <div class="container">
-                            <t t-set="providers" t-value="env['product.brand'].search([('authorized_partner_ids', '!=', False)], order='name desc')"/>
+                            <t t-set="providers" t-value="env['product.brand'].sudo().search([('authorized_partner_ids', '!=', False)], order='name desc')"/>
```

Comment:
@JulioSerna @nhomar 
Should we use `sudo` or should we use a new ACL to allow read for portal users?